ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/pipeline/exception/PipelineInitialisationException.java

```diff
@@ -0,0 +1,16 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.pipeline.exception;
+
+public class PipelineInitialisationException extends PipelineException {
+	private final String pipelineName;
+	private final Exception exception;
+
+	public PipelineInitialisationException(String pipelineName, Exception exception) {
```

Comment:
I would rather like to see the abstract super class also implement the `super(Throwable cause)` constructor and pass this exception through as the cause